composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "payone-gmbh/woocommerce-3",
+    "description": "PAYONE Payment Integration for WooCommerce.",
+    "type": "wordpress-plugin",
+    "keywords": [],
+    "homepage": "https://www.payone.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "PAYONE",
+            "email": "integrations@payone.com",
+            "homepage": "https://www.payone.com"
+        }
+    ],
+    "support": {
+        "email": "tech.support@payone.com",
+        "issues": "https://github.com/PAYONE-GmbH/woocommerce-3/issues",
+        "docs": "https://docs.payone.com"
```

Comment:
maybe link to the actual doc: https://docs.payone.com/display/public/INT/WooCommerce+Plugin